packages/frontend/core/src/blocksuite/ai/provider/copilot-client.ts

```diff
@@ -40,18 +39,13 @@ function codeToError(error: UserFriendlyError) {
       return new GeneralNetworkError(
         error.code
           ? `${error.code}: ${error.message}\nIdentify: ${error.name}`
-          : undefined
+          : error.message
       );
   }
 }
 
 export function resolveError(err: any) {
-  const standardError =
-    err instanceof GraphQLError
-      ? new UserFriendlyError(err.extensions)
-      : UserFriendlyError.fromAnyError(err);
-
-  return codeToError(standardError);
+  return codeToError(err);
```

Comment:
looks will lost error info in GraphQLError

<img width="514" alt="image" src="https://github.com/user-attachments/assets/0a375adc-5f1e-4375-8ecf-95c040522123" />

Reply:
forget it, rebase #10667